### PR TITLE
Fix ets to not read schema

### PR DESCRIPTION
**Pull request by YanfeiNREL**

### CODE REVIEW

**Comment by YanfeiNREL** [comment]
Hey Nick, please review and merge this branch, if there is no issue here. It is also the basis for me to revise another two branches. @nllong

File: geojson_modelica_translator/system_parameters/system_parameters.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def get_param_by_building_id(self, building_id, path, default=None):
         # This will get reworked after moving to jsonpath. but for now, hack in the default. First return the default
         # dict from the system parameter file.
         # Grab first the default data block, then find the path in the default data block.
-        default_data = self.get_param(f'buildings.default', impute_default=False)
+        default_data = self.get_param(f"{'buildings.default'}", impute_default=False)
```
Comment (nllong) [inline geojson_modelica_translator/system_parameters/system_parameters.py:174]: this looks strange. I think it should just be `'buildings.default'`. Remove the `f` in the original code.

File: geojson_modelica_translator/model_connectors/ets_template.py [hunk 2]
```diff
@@ -69,66 +49,35 @@ def __init__(self, thermal_junction_properties_geojson, system_parameters_geojso
...
-        with open(path_ets_templated, "w") as f:
-            f.write(file_data)
-
-        # write templated ETS back to building-modelica folder for Dymola test
-        path_writtenback = os.path.join(os.path.abspath(os.path.join(__file__, "../..")) + "/modelica/")
+        ets_template = self.template_env.get_template("CoolingIndirect.mot")
+        b_modelica_path = ModelicaPath(
+            f"B{building['building_id']}", scaffold.loads_path.files_dir, True
+        )
+        ets_model_type = self.system_parameters.get_param_by_building_id(
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/ets_template.py:63]: this is better, thanks!

File: geojson_modelica_translator/model_connectors/ets_template.py [hunk 3]
```diff
@@ -28,39 +28,19 @@
 ****************************************************************************************************
 """
 
-import json
+# import json
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/ets_template.py:31]: remove if not needed.

File: geojson_modelica_translator/system_parameters/system_parameters.py (hunk 1 above)
Comment (YanfeiNREL) [reply geojson_modelica_translator/system_parameters/system_parameters.py]: it failed here using 'buildings.default'.

**Comment by nllong** [comment]
Not sure what is going on with travis. I cleared the cache and restarted the jobs. The fix with NOT f"{'building.default'}" worked fine locally.

**Pull request merged**